### ISSUE TITLE
Multi-chart graph. x-axis alighnment issue

### ISSUE THREAD

**Issue by rajaron** [novus/nvd3#1989]
Hi, 

Developed a multi-chart graph. x-axis label and bar chart are not aligning. Can any body help on this

![chart](https://cloud.githubusercontent.com/assets/3753490/23849495/cef732ba-0801-11e7-9394-043bc03ea756.png)

Regards,
~ Rajkumar

**Comment by bgth** [comment]
Hi, Can you please share a jsfiddle for the issue?

**Comment by deepakjan** [comment]
Have a similar issue posted earlier also and below is the jsfiddle link for the same thanks 


[http://jsfiddle.net/n2hfN/76/](url)

**Comment by deepakjan** [comment]
[http://jsfiddle.net/n2hfN/76/](url)

http://jsfiddle.net/n2hfN/76/

**Comment by bgth** [comment]
Hi @deepakjan, The jsfiddle you gave is working fine.
![screenshot](https://cloud.githubusercontent.com/assets/7175235/23929685/d1bca53c-094d-11e7-92f6-604aba80680e.jpg)

**Comment by anthonysubramani** [comment]
Hi @bgth @deepakjan, this fiddle provides the alignment issue

https://jsfiddle.net/ctdsjopk/3/

**Comment by bgth** [comment]
Yeah.. I can see that now. As it is coded now, it is taking the position of the line chart x axis.

**Comment by deepakjan** [comment]
So is there a solution for this?  to make bar chart also aligned with X-axis? in the multi charts
